.github/workflows/build.yml

```
@@ -0,0 +1,25 @@
+name: ble.sh build
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  builds:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          submodules: recursive
+      - name: Package
+        run: |
+          make install INSDIR="$(pwd)/ble.sh"
+          tar -cJf "ble-nightly.tar.xz" ble.sh
```

Ah, sorry what I originally meant was something like

```suggestion
          make install INSDIR="$(pwd)/ble-nightly"
          tar -cJf "ble-nightly.tar.xz" ble-nightly
```

I would like to also make the directory name and the archive name consistent with each other. But, let's decide it after we conclude how the distributed versions should be managed.